src/amqp10/encoder.ts

```diff
@@ -34,14 +35,21 @@ const PropertySizeDescription =
 
 type MessageApplicationPropertiesList = [string, string | number][]
 
-export function amqpEncode(writer: DataWriter, { content, messageProperties, applicationProperties }: Message): void {
+type MessageAnnotationsList = [string, string | number][]
```

Comment:
I don't have seen this list of list before why don't create something more usefull like:
```
{key: string, value: string| number}[]
```
if you agree I suggest to refactor also the `MessageApplicationPropertiesList`